tests/testMinimize.py

```diff
@@ -72,22 +72,19 @@ def testMinimize2(self):
 
         print("modelParams=", modelParams)
         print("fitParams  =", fitResults.parameterList)
-        self.assert_(fitResults.isValid, "fit failed")
+        self.assertTrue(fitResults.isValid, "fit failed")
         if not numpy.allclose(modelParams, fitResults.parameterList):
```

Comment:
I think that fitsResults.isValid is a boolean that gives that state of the fit. The "fit failed" was a message used by assert_ to give the reason for the failure. I think this is ok as is, do you agree or am I missing something?